### PR TITLE
refactor: refine create factory contract

### PR DESCRIPTION
**Pull request by ksin751119**
- rename `@author`
- remove address parameter of `getDeployed()`
- add `deployer` parameter in constructor()

### CODE REVIEW

File: src/CREATE3Factory.sol [hunk 1]
```diff
@@ -9,25 +9,25 @@ import {Ownable} from "openzeppelin-contracts/contracts/access/Ownable.sol";
 
 /// @title Factory for deploying contracts to deterministic addresses via CREATE3
 /// @author zefram.eth
```
Comment (hihiben) [inline src/CREATE3Factory.sol:11]: Should remove?

File: src/ICREATE3Factory.sol [hunk 2]
```diff
@@ -3,26 +3,20 @@ pragma solidity >=0.6.0;
 
 /// @title Factory for deploying contracts to deterministic addresses via CREATE3
 /// @author zefram.eth
```
Comment (hihiben) [inline src/ICREATE3Factory.sol:5]: Same.

File: src/CREATE3Factory.sol (hunk 1 above)
Comment (ksin751119) [reply src/CREATE3Factory.sol:11]: Should we keep the `@author zefram.eth` because of the AGPL-3.0 License?

File: src/ICREATE3Factory.sol (hunk 2 above)
Comment (ksin751119) [reply src/ICREATE3Factory.sol:5]: Should we keep the `@author zefram.eth` because of the AGPL-3.0 License?

File: src/CREATE3Factory.sol (hunk 1 above)
Comment (hihiben) [reply src/CREATE3Factory.sol:11]: I think we do but i'm not sure if this is the right way to do that. If so then i have no additional questions.

File: src/ICREATE3Factory.sol [hunk 3]
```diff
@@ -3,26 +3,20 @@ pragma solidity >=0.6.0;
...
-        external
-        payable
-        returns (address deployed);
+    function deploy(
+        bytes32 salt,
+        bytes memory creationCode
+    ) external payable returns (address deployed);
 
     /// @notice Predicts the address of a deployed contract
     /// @dev The provided salt is hashed together with the deployer address to generate the final salt
```
Comment (zodahu) [inline src/ICREATE3Factory.sol]:
```suggestion
    /// @dev The provided salt is hashed to generate the final salt
```

File: README.md
Comment (zodahu) [inline README.md:1]: > the address of the deployed contract is determined by only the deployer address and the salt.

the address of the deployed contract is determined by only the salt.

Comment (zodahu) [inline README.md:1]: > The salt provided is hashed together with the deployer address (i.e. msg.sender) to form the final salt, such that each deployer has its own namespace of deployed addresses.

Remove this line

**Comment by zodahu** [comment]
LGTM

**Review by hihiben — APPROVED** [review]
LGTM

**Pull request merged**